codescan/responses.go

```diff
@@ -185,6 +185,7 @@ func (r *responseBuilder) buildFromField(fld *types.Var, tpe types.Type, typable
 		if err := sb.buildFromType(ftpe.Elem(), schemaTypable{schema, typable.Level() + 1}); err != nil {
 			return err
 		}
+		r.postDecls = append(r.postDecls, sb.postDecls...)
```

Comment:
Thank you for digging this out.
Would it be possible to assert the fix with some unit test case?

Reply:
I just add some test code in `codescan/responses_test.go` and some file in fixtures/goparsing for test case . 
but I have no idea why after submitting this commit will cause somewhere fails in `go test -v -timeout 60m hack/codegen_nonreg_test.go -args -fixture-file codegen-fixtures.yaml -skip-models -skip-full-flatten`